docs/settings.rst

```diff
@@ -9,13 +9,20 @@ Settings reference
 
 Here is a complete list of settings which control default PynamoDB behavior.
 
+connect_timeout_seconds
+-----------------------
+
+Default: ``60``
+
+The time till a timeout exception is thrown when attempting to make a connection.
+
 
-request_timeout_seconds
+read_timeout_seconds
 -----------------------
 
 Default: ``60``
 
-The default timeout for HTTP requests in seconds.
+The time till a timeout exception is thrown when attempting to read from a connection.
```

Comment:
```suggestion
The time in seconds until a URLError is raised when attempting to read from a connection.
```
(assuming it's `URLError`)

Reply:
I think these: https://github.com/boto/botocore/blob/cd7fd229c4dd3637bceac440a0b31a142774754a/botocore/exceptions.py#L109

Reply:
Either one?